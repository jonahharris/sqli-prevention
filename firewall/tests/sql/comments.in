>>> Sada testovacich SQL dotazu: komentare
SELECT * FROM authors; # Toto je komentar
SELECT * FROM authors; -- Toto je komentar
SELECT * FROM authors; /* Toto je \nviceradkovy \nkomentar */
select 1 /* + 1 */;
