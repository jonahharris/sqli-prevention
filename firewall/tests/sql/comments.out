>>> Sada testovacich SQL dotazu: komentare
SELECTALL;TABLE authors;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE authors;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE authors;SELECT 0 1 1;STMT;1;0;
NUMBER 1;SELECTNODATA 0 1;STMT;1;0;
